### DISCUSSION TITLE
I found  a  issue  in netty guide

### DISCUSSION THREAD

**Discussion by BruceBie** [netty/netty#11947] [Q&A]
https://netty.io/wiki/user-guide-for-4.x.html
![nettyproblem](https://user-images.githubusercontent.com/31531594/147212776-37889cd1-5882-45e2-8b72-1a3f9b8817b5.png)

rdate command:
$ rdate -o <port> -p <host>

it  is  wrong  in  -o   ,this is not  -o option  in  rdate

**Comment by chrisvest** [comment]
Looks like there are a number of conflicting versions around.

The Debian version has the `-o` option: https://manpages.debian.org/testing/rdate/rdate.8.en.html
But MacOS/Homebrew gives you this one: https://www.aelius.com/njh/rdate/
And Fedora gives you OpenRdate, which in recent versions also has the `-o` option, except Fedora uses a version 1.2 from 2007: https://github.com/resurrecting-open-source-projects/openrdate

Feel free to work through the example and see if you can make it work across relevant platforms.
The user guide on the website is pulled from our wiki: https://github.com/netty/netty/wiki/User-guide-for-4.x
